meson.build

```diff
@@ -571,16 +571,38 @@ if build_standalone
   subdir('po')
   subdir('libfwupdplugin')
   subdir('contrib')
-
-  # the plugins each add the gresource files into an array, and then in src:
-  # * for -Dgresource_quirks=enabled, they get built into ./src/fwupd and the files are ignored
-  # * for -Dgresource_quirks=disabled, the files get installed into quirks.d
   subdir('plugins')
   subdir('src')
-
 endif
 subdir('docs')
 
+# install the quirk files
+if get_option('compressed_quirks')
```

Comment:
I think you need to fail or disable the feature if gzip command not found.

Reply:
We already have a hard requirement of `find_program('gzip')` for `host-emulate.d` -- and tbh, i think gzip is a just fine hard dep.